libpathod/pathod.py

```diff
@@ -255,7 +265,7 @@ def handle_http_request(self):
                     path=path,
                     method=method,
                     headers=headers.lst,
-                    httpversion=httpversion,
```

Comment:
Intentional? (According to the comment, this method should support HTTP 1 as well)